src/cargo/util/command_prelude.rs

```diff
@@ -405,11 +405,9 @@ pub trait ArgMatchesExt {
     }
 
     fn index(&self, config: &Config) -> CargoResult<Option<String>> {
-        // TODO: Deprecated
-        // remove once it has been decided --host can be removed
-        // We may instead want to repurpose the host flag, as
-        // mentioned in this issue
-        // https://github.com/rust-lang/cargo/issues/4208
+        // FIXME: deprecated. Remove once it has been decided `--host` can be removed
```

Comment:
Any reason this was "upgraded" to FIXME?  (mostly just curious)

Reply:
I was just used to using `FIXME` everywhere, and not `TODO`, which is very rare in the Rustc codebase. I've reverted though (not that it matters much eh).